src/Microsoft.ApplicationInsights.AspNetCore/project.json

```diff
@@ -21,83 +21,83 @@
+            "type": "build"
+        },
+        //"System.Security.Cryptography.Hashing.Algorithms.Analyzers": {
+        //    "version": "1.2.0-beta2",
+        //    "type": "build"
+        //},
+        "System.Threading.Tasks.Analyzers": {
+            "version": "1.2.0-beta2",
+            "type": "build"
+        }
+        //"Text.Analyzers": {
+        //    "version": "1.2.0-beta2",
+        //    "type": "build"
+        //},
+        //"XmlDocumentationComments.Analyzers": {
+        //    "version": "1.2.0-beta2",
+        //    "type": "build"
+        //},
     },
-    "Microsoft.ApiDesignGuidelines.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "Microsoft.ApplicationInsights": "2.1.0",
-    "Microsoft.AspNetCore.Hosting.Abstractions": "1.0.0",
-    "Microsoft.AspNetCore.Http.Abstractions": "1.0.0",
-    "Microsoft.AspNetCore.Mvc.ViewFeatures": "1.0.1",
-    "Microsoft.CodeAnalysis.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "Microsoft.Composition.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "Microsoft.Extensions.Configuration": "1.0.0",
-    "Microsoft.Extensions.DiagnosticAdapter": "1.0.0",
-    "Microsoft.Extensions.Logging.Abstractions": "1.0.0",
-    "Microsoft.Maintainability.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "Microsoft.QualityGuidelines.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "Roslyn.Diagnostics.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "System.Collections.Immutable.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    //"System.Resources.Analyzers": {
-    //    "version": "1.2.0-beta2",
-    //    "type": "build"
-    //},
-    "System.Runtime.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    "System.Runtime.InteropServices.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    },
-    //"System.Security.Cryptography.Hashing.Algorithms.Analyzers": {
-    //    "version": "1.2.0-beta2",
-    //    "type": "build"
-    //},
-    "System.Threading.Tasks.Analyzers": {
-        "version": "1.2.0-beta2",
-        "type": "build"
-    }
-    //"Text.Analyzers": {
-    //    "version": "1.2.0-beta2",
-    //    "type": "build"
-    //},
-    //"XmlDocumentationComments.Analyzers": {
-    //    "version": "1.2.0-beta2",
-    //    "type": "build"
-    //},
-  },
 
   "frameworks": {
     "net451": {
-      "dependencies": {
-        "Microsoft.ApplicationInsights.DependencyCollector": "2.1.0",
-        "Microsoft.ApplicationInsights.PerfCounterCollector": "2.1.0",
-        "Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel": "2.1.0"
-      }
+        "dependencies": {
+            "Microsoft.ApplicationInsights.DependencyCollector": "2.2.0-beta3",
```

Comment:
since you've added pre-release dependency here - the version of package itself should be pre-release. I don't see the change that will mark it pre-release